src/app/app.module.ts

```diff
@@ -52,10 +66,22 @@ import { TalkingHeadsMapComponent } from './templates/talking-heads/talking-head
     MainPageComponent,
     TimelineMapComponent,
     TalkingHeadsMapComponent,
+    PrevDirective,
+    NextDirective,
+    M6Component,
+    SeclayoutComponent,
+    AccordionDirective,
+    SynchDirective,
+    PaginationDirective,
+    InfoComponent,
```

Comment:
Perhaps it would be a good idea to give your components some prefix if their names are too generic (e.g. this one...)